depends/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ option(CLSOCKET_DEP_ONLY "Build for use inside other CMake projects as dependenc
 add_subdirectory(clsocket)
 ide_folder(clsocket "Depends")
 
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/luacov/src/luacov/ DESTINATION ${DFHACK_DATA_DESTINATION}/lua/luacov)
```

Comment:
I'll add a check to make sure the submodule exists with the other checks, since that can trip up people with existing clones sometimes.